<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userMange</class>
 <widget class="QWidget" name="userMange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1111</width>
     <height>671</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> color:rgb(255,215,0);  
    min-height:20;  
    border-style:solid;  
    border-top-left-radius:2px;  
    border-top-right-radius:2px;  
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 rgb(226,236,241),   
                                stop: 0.3 rgb(33,184,150),  
                                  stop: 1 rgb(29,170,140));  </string>
   </property>
   <property name="text">
    <string>按照用户类型查找</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	width:142px;
	height:40px;
	line-height:18px;
	color:rgb(85, 0, 0);	
	background:url(&quot;:/mybutton/button/bg35.jpg&quot;) no-repeat left top;	
	padding-bottom:4px;</string>
   </property>
   <property name="text">
    <string>按照ID查找</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_searchid">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_search">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	width:142px;
	height:40px;
	line-height:18px;
	color:rgb(85, 0, 0);	
	background:url(&quot;:/mybutton/button/bg35.jpg&quot;) no-repeat left top;	
	padding-bottom:4px;</string>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_adduser">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>10</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> color:rgb(255,215,0);  
    min-height:20;  
    border-style:solid;  
    border-top-left-radius:2px;  
    border-top-right-radius:2px;  
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 rgb(226,236,241),   
                                stop: 0.3 rgb(33,184,150),  
                                  stop: 1 rgb(29,170,140));  
    border:1px;  
    border-radius:5px;padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>添加用户</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
